app/models/spree/spree_stock_movement_decorator.rb

```diff
@@ -0,0 +1,9 @@
+module Spree
+  module StockMovementDecorator
+    def self.prepended(base)
+      base.belongs_to :spree_stock_movement_reason
```

Comment:
Por defecto en rails5+ los belongs to son obligatorios, si queda así hay otros lugares donde se podría caer. Hay que especificar que sea opcional y en la vista de stock_items lo obligamos

Reply:
Sería con un optional: true?